task/golang-build/README.md

```diff
@@ -0,0 +1,55 @@
+# Golang Build
+
+This Task is Golang task to build Go projects.
+
+## Install the task
+
+TODO
+
+
+### Parameters
+
+* **package**: base package under test
+* **packages**: packages to test (_default:_ ./...)
+* **version**: golang version to use for tests (_default:_ latest)
+* **flags**: flags to use for `go build` command (_default:_ -v)
+* **GOOS**: operating system target (_default:_ linux)
+* **GOARCH**: architecture target (_default:_ amd64)
+* **GO111MODULE**: value of module support (_default:_ auto)
+* **GOCACHE**: value for go caching path (_default:_ "")
+* **GOMODCACHE**: value for go module caching path (_default:_ "")
+* **CGO_ENABLED**: value for go cgo tool toggle (_default:_ "")
+* **GOSUMDB**: value for go checksum database url (_default:_ "")
+
+### Workspaces
+
+* **source**: A [Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md) containing the source to build.
+
+### Platforms
+
+The Task can be run on `linux/amd64`, `linux/s390x`,  and `linux/ppc64le` platforms.
+
+Specify value for `GOARCH` parameter according to the desired target architecture.
+
+## Usage
+
+This TaskRun runs the Task to compile commands from
+[`tektoncd/pipeline`](https://github.com/tektoncd/pipeline).
+`golangci-lint`.
```

Comment:
I guess we can remove `golangci-lint` from here

Reply:
Thanks @vinamra28! Removed.